src/actions/LocalSettingsActions.ts

```diff
@@ -154,8 +157,8 @@ export const writeAccountNotifInfo = async (
   notifInfo: Partial<NotifInfo>
 ): Promise<LocalAccountSettings> => {
   return await writeAccountNotifState(account, {
-    ...localAccountSettings.notifState,
-    [accountNotifStateKey]: { ...(localAccountSettings.notifState[accountNotifStateKey] ?? asNotifInfo({})), ...notifInfo }
+    ...getLocalAccountSettings().notifState,
+    [accountNotifStateKey]: { ...(getLocalAccountSettings().notifState[accountNotifStateKey] ?? asNotifInfo({})), ...notifInfo }
```

Comment:
For this and the following `write` methods, I would suggest:

```ts
const settings  = getLocalAccountSettings()
return await writeAccountNotifState(account, {
  ...settings,
  thingToUpdate: {...settings.thingToUpdate, newItem }
})
```

As opposed to repeated calls to `getLocalAccountSettings()`